### ISSUE TITLE
[Bug]: 9.4.1 Dev build QT errors.

### ISSUE THREAD

**Issue by myxxmikeyxx** [TagStudioDev/TagStudio#514]
### Checklist

- [X] I am using an up-to-date version.
- [X] I have read the [documentation](https://github.com/TagStudioDev/TagStudio/blob/main/docs/index.md).
- [X] I have searched existing [issues](https://github.com/TagStudioDev/TagStudio/issues).

### TagStudio Version

Alpha 9.4.1 Dev

### Operating System & Version

MacOS 14.2.1 (Intel)

### Description

Getting an error when I launch TagStudio on the Mac form the terminal
```
No QtMultimedia backends found. Only QMediaDevices, QAudioDevice, QSoundEffect, QAudioSink, and QAudioSource are available.
Failed to create QVideoSink "Not available"
Failed to initialize QMediaPlayer "Not available"
```
Bug Report is attached below and the change log from 6.7.0 -> 6.7.1 is listed. two CVE's were listed but they are not directly used by QT and only used if the developer is using them. I did not check to see if the code contains, but since you are trying to use sqlite DB and are not networking this should be in the clear.

https://bugreports.qt.io/browse/PYSIDE-2656
https://code.qt.io/cgit/qt/qtreleasenotes.git/about/qt/6.7.1/release-note.md
https://forum.qt.io/topic/157026/no-qtmultimedia-backends-found-on-macos-with-pyside-6-7-1

I created a pull request if you think this is worth fixing.
https://github.com/TagStudioDev/TagStudio/pull/513

### Expected Behavior

No QT error

### Steps to Reproduce

Be on MacOS 14.2.1
Download the source code Zip file
Run and let it install requirements

**Comment by myxxmikeyxx** [comment]
Created pull request. #513

**Comment by myxxmikeyxx** [comment]
This seems to be fixed if the pull request for #557 is merged.
